src/megaapi_impl.cpp

```diff
@@ -23107,6 +23148,128 @@ void MegaApiImpl::sendPendingRequests()
                                                            request->getNodeHandle()));
             break;
         }
+        case MegaRequest::TYPE_BACKUP_FOLDER:
+        {
+            // validate the request
+            if (!request->getFile() || !request->getName()) { e = API_EACCESS; break; }
+
+            // get current user
+            User* u = client->ownuser();
+
+            // get handle of remote "My Backups" folder, from user attributes
+            if (!u || !u->isattrvalid(ATTR_MY_BACKUPS_FOLDER)) { e = API_EACCESS; break; }
+            const string* handleContainerStr = u->getattr(ATTR_MY_BACKUPS_FOLDER);
+            if (!handleContainerStr) { e = API_EACCESS; break; }
+
+            std::unique_ptr<TLVstore> tlvRecords(TLVstore::containerToTLVrecords(handleContainerStr, &client->key));
+            if (!tlvRecords || !tlvRecords->find("h")) { e = API_ENOENT; break; }
+
+            const string& handleStr = tlvRecords->get("h");
+            handle h = MegaApi::base64ToHandle(handleStr.c_str());
+            if (h == UNDEF) { e = API_ENOENT; break; }
+
+            // get Node of remote "My Backups" folder
+            MegaNode* myBackupsNode = getNodeByHandle(h);
```

Comment:
You may want to use a more efficient approach by using a `Node` instead of `MegaNode` (simply need to call `client->nodebyhandle(h)` ;)